### ISSUE TITLE
Staging Review finding: Focus lost when uploading supporting evidence

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#122410]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Design System & Forms Library
**Product:** Form 21-4140
**Feature:** Employment Questionnaire

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When using keyboard alone, uploading a document after attaching, focus resets to the top of the page. This can be disorienting for users including keyboard only users and screen reader users. 

<details><summary>Link, screenshot or steps to recreate</summary>

- Using keyboard only
- Navigate to Step 5 of 6 Upload supporting evidence
- TAB to the Upload your supporting evidence file upload
- Activate using Enter
- Select a file and attach
- Note where focus is when the system window closes and you TAB

Note: Only able to recreate in Safari with Voiceover

</details>

## Recommended action

Ensure focus is actively managed, and focus returns to a logical location. A logical location in this component would be on the file name, or similar, so that the user is aware that the file attached successfully.  

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
